decidim-accountability/app/packs/entrypoints/decidim_accountability_admin_imports.js

```diff
@@ -0,0 +1,3 @@
+import initializeElement from "src/decidim/accountability/admin/imports"
+
+window.addEventListener("load", initializeElement());
```

Comment:
Could you use the "DOMContentLoaded" event instead of "load" here?

```suggestion
window.addEventListener("DOMContentLoaded", initializeElement());
```

It is safer in such cases as this one, as the "load" event requires that the related nodes are already available in the DOM when this script is loaded. It is the case right now but in case this script is e.g. moved to the `<head>` of the page, it would break its functionality.